code/_core/datum/damagetype/ranged/bullet/shotgun_23.dm

```diff
@@ -40,4 +40,27 @@
 		PIERCE = DAMAGE_GREATSWORD*0.1
 	)
 
-	falloff = VIEW_RANGE
\ No newline at end of file
+	falloff = VIEW_RANGE
+
+/damagetype/ranged/bullet/shotgun_23/fire
+	name = "spicy pellet"
+
+	//The base attack damage of the weapon. It's a flat value, unaffected by any skills or attributes.
+	attack_damage_base = list( //so uuh around 90% of regular damage?
+		BLUNT = DAMAGE_GREATSWORD*0.5,
+		HEAT = DAMAGE_GREATSWORD*0.4
```

Comment:
Damage should equal one whole of a define.

Reply:
After a single test I got really concerned whether I should or shouldn't add incendiary 23x75, since it's quite powerful. Anyways, I'll fix that.